Assets/Scripts/UIElements/InteractionIcon.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class InteractionIcon : MonoBehaviour
+{
+    
+    public Texture reflectionImg;
+    public Texture handImg;
+    public Texture eyeImg;
+    public RawImage iconImg;
+    public GameObject interactionIcon;
+    
+    void Start()
+    {
+        interactionIcon.SetActive(false);
+    }
+    public void showReflectionIcon()
```

Comment:
Same style error with function names in this script as well. Function names should start with capital letter